repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/opentracing/httpclient/ReposeTracingResponseInterceptor.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.commons.utils.opentracing.httpclient
+
+import java.io.IOException
+
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import io.opentracing.Span
+import io.opentracing.tag.Tags._
+import org.apache.http._
+import org.apache.http.protocol.HttpContext
+
+/**
+  * An [[org.apache.http.HttpResponseInterceptor]] that will enrich Repose HTTP requests made through an
```

Comment:
"requests" -> "responses".

Reply:
I was taking this from the perspective that it was still the request that was the cause of the enrichment, but yeah. Whatever. Updated.